src/main/java/org/ambraproject/wombat/freemarker/ReplaceParametersDirective.java

```diff
@@ -96,15 +97,8 @@ public void execute(Environment environment, Map params, TemplateModel[] loopVar
 
     // The map is passed in as a Map<String, String[]>, but Freemarker doesn't support generics
```

Comment:
Update comment.

Reply:
I think this is still an accurate comment, see https://github.com/apache/tomcat85/blob/b21d02042c408053a1944b27c3196ec9be3db381/java/org/apache/catalina/connector/RequestFacade.java#L432

Reply:
I tried for a few hours to figure out why this is happening, but I can't figure it. There is some change between tomcat versions that causes freemarker to pass in a different type as the parameterMap. Frankly all this code is not great: the `replaceParameters` directive is a hack, and we are implementing our own access to the the request parameters to work around a "bug" that probably doesn't effect us (https://github.com/plos/wombat/blob/AMBR-886/src/main/java/org/ambraproject/wombat/controller/SearchController.java#L326).